pkg/test/step.go

```diff
@@ -139,7 +139,7 @@ func (s *Step) DeleteExisting(namespace string) error {
 	}
 
 	// Wait for resources to be deleted.
-	return wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (done bool, err error) {
+	return wait.PollImmediate(100*time.Millisecond, time.Duration(s.GetTimeout())*time.Second, func() (done bool, err error) {
```

Comment:
I'm pretty sure s.GetTimeout returns a value in seconds, so we need to multiply before creating the duration (as that's in nanoseconds)
```suggestion
	return wait.PollImmediate(100*time.Millisecond, time.Duration(s.GetTimeout()*time.Second), func() (done bool, err error) {
```

Reply:
This won’t compile. We multiply after converting the integer to Duration already.

Reply:
Indeed.  The go way is weird sometimes. :)